packages/react-magma-dom/src/components/Table/Table.tsx

```diff
@@ -122,6 +128,12 @@ export const TableContainer = styled.div<{
           : props.theme.colors.focus};
   }
 `;
+export const StyledCaption = styled.caption<{ isInverse: boolean; tableTitleNode: boolean; }>`
```

Comment:
By default, captions are centered aligned.

@orion-cengage are you okay with this or should we force a different alignment?
![image](https://github.com/user-attachments/assets/3d90a784-fac0-4119-ad80-12ec9dafd602)